### PR TITLE
Adding isbnlib

### PR DESCRIPTION
**Pull request by maxb2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maxb2** [comment]
I'm willing to be listed as a recipe maintainer.

**Comment by maxb2** [comment]
@conda-forge/help-python, ready for review!

File: recipes/isbnlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256: c9e6c1dcaa9dff195429373cf2beb3117f30b3fca43d7db5aec5a2d1f6f59784
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3
```
Comment (BastianZim) [inline recipes/isbnlib/meta.yaml:19]:
```suggestion
    - python >=3.6
```

But please double-check that.

Ref: https://github.com/xlcnd/isbnlib/blob/cd24d6af137d8c9ee95d75a3fdfe8377552f4a10/setup.cfg#L11

File: recipes/isbnlib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - pip
+  run:
+    - python >=3
```
Comment (BastianZim) [inline recipes/isbnlib/meta.yaml:22]:
```suggestion
    - python >=3.6
```

File: recipes/isbnlib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/xlcnd/isbnlib
+  summary: Extract, clean, transform, hyphenate and metadata for ISBNs (International Standard Book Number).
+  license: LGPL-3.0-or-later
+  license_file: LICENSE-LGPL-3.0.txt
```
Comment (BastianZim) [inline recipes/isbnlib/meta.yaml:36]:
```suggestion
  license_file: 
    - LICENSE-LGPL-3.0.txt
    - COPYRIGHT.txt
```
This needs to be included as well.

**Review by BastianZim — APPROVED** [review]

**Pull request merged**